stages/eib_image

```diff
@@ -597,6 +597,22 @@ EOF
     "${EIB_HELPERSDIR}"/create-vm-image "$(eib_outfile ovf.zip)" "${img}"
   fi
 
+  # Create qcow2 image if required.
+  if [ "${EIB_IMAGE_QCOW2}" = true ]; then
+    local qcow2
+
+    qcow2=$(eib_outfile qcow2)
+    qemu-img convert -c -f raw -O qcow2 "${img}" "${qcow2}"
+
+    # Since the format is copy-on-write, we can grow it as much as we
+    # want without increasing the download size. Add 10 GiB.
+    qemu-img resize -f qcow2 "${qcow2}" +10G
```

Comment:
OK this addresses my first reflexive reaction: whenever I convert an image to qcow2 manually (which I do relatively frequently!) I forget to do this and it's really annoying to go back and do it later.

Reply:
One thing I've always wanted is a simpler way to trigger the initramfs partitioner to run again to support this kind of after the fact VM resizing.

Reply:
If you're "in the know", you can add the [marker attribute](https://github.com/endlessm/eos-boot-helper/blob/master/dracut/repartition/endless-repartition.sh#L99) back to the partition so it's resized on the next boot. You also need to delete [`/var/eos-booted`](https://github.com/endlessm/eos-boot-helper/blob/master/eos-firstboot.service#L8) or manually run `resize2fs` after that. But it would be a lot more convenient if there was a tool do that.